src/identity/components/GlobalHeader/GlobalHeaderDropdown/GlobalHeaderDropdown.scss

```diff
@@ -51,9 +51,19 @@
   background-color: rgba(241, 241, 243, 0.1);
 }
 
-.global-header--align-center {
```

Comment:
I changed the name here because this element was really being used as a style for the global header main dropdown items.